fedimint-api/src/config.rs

```diff
@@ -174,14 +231,14 @@ impl ClientModuleConfig {
 pub struct ServerModuleConfig {
     pub local: serde_json::Value,
     pub private: serde_json::Value,
-    pub consensus: serde_json::Value,
+    pub consensus: JsonWithKind,
```

Comment:
Well... in certain places we pass only a part (as we split and assemble them when dealing with modules and their calling conventions). 

No strong feelings about it. I did it here, because compiler told me we need it. :D

I think I rather have the kind in all of them, just in case someone needs them, and not have to generalize all the version of configs (typed, erased, split, assembled)...